docs/user-guide.md

```diff
@@ -43,6 +43,7 @@ published: True
       - [RAC configuration parameters](#rac-configuration-parameters)
       - [Backup configuration parameters](#backup-configuration-parameters)
       - [Additional operational parameters](#additional-operational-parameters)
+      - [Using tags to run or skip specific parts of the toolkit](#using-tags-to-run-or-skip-specific-parts-of-the-toolkit)
```

Comment:
Thanks for putting this together;  unless it's automated, I'd actually prefer not to exhaustively list tags in the documentation, as they do change quite regularly and would be extra toil to maintain;  I'd suggest someone wanting to see the individual tags should just look at the code itself.